nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -1063,7 +1063,7 @@ Random starsystem. Excluded from starmap saving, as they're generated at init.
 
 /datum/star_system/proc/generate_badlands()
 	var/list/generated = list()
-	var/amount = rand(50, 70)
+	var/amount = rand(17, 25)
```

Comment:
I know this seems very drastic but it didn't look bad when I opened it up on my test server I promise

Reply:
I've been waiting for this so we can actually do more Armada mode stuff.